### DISCUSSION TITLE
v1.9.4

### DISCUSSION THREAD

**Discussion by GeckoEidechse** [R2Northstar/Northstar#325] [Release]
**Features:**
Allow comments and newlines in `banlist.txt` ([#227](https://github.com/R2Northstar/NorthstarLauncher/pull/227))
Fixed unban not overwriting `banlist.txt` ([#228](https://github.com/R2Northstar/NorthstarLauncher/pull/228))
Hot-reload banlist on player join ([#233](https://github.com/R2Northstar/NorthstarLauncher/pull/233))

**Fixes:**
Fix most clang/mingw issues ([#226](https://github.com/R2Northstar/NorthstarLauncher/pull/226))
Adding win achievement ([#481](https://github.com/R2Northstar/NorthstarMods/pull/481))
Swap `EVAC_WAIT_TIME` to `waitTime` in `_evac.gnut` ([#482](https://github.com/R2Northstar/NorthstarMods/pull/482))

**Contributors:**
@catornot @pg9182 @ScureX @x3Karma

<hr /><em>This discussion was created from the release <a href='https://github.com/R2Northstar/Northstar/releases/tag/v1.9.4'>v1.9.4</a>.</em>

**Comment by Malik12097** [comment]
Help me to get the northstar client Download it

**Comment by Malik12097** [comment]
On Xbox

**Reply by GeckoEidechse** [reply]
Northstar is not supported on Xbox and never will be, simply due to the restrictive nature of consoles. Consoles do not allow modding in general and as such, that also includes Northstar.

**Reply by Malik12097** [reply]
Can I get it

**Reply by Malik12097** [reply]
Can you send the link for get it

**Reply by GeckoEidechse** [reply]
To repeat myself. No Northstar does NOT work on console and we won't add support for it either, simply has that's a task way beyond our capabilities.

**Reply by Malik12097** [reply]
Because I like TitanFall 2

**Reply by GeckoEidechse** [reply]
For PC, check https://r2northstar.gitbook.io/r2northstar-wiki/installing-northstar/basic-setup

**Comment by Malik12097** [comment]


**Reply by Malik12097** [reply]
Send me the video how to get it

**Comment by Malik12097** [comment]
Help me
